src/cryptoadvance/specter/server.py

```diff
@@ -26,6 +27,26 @@
 csrf = CSRFProtect()
 
 
+class SpecterFlask(Flask):
```

Comment:
That looks interesting. I think this is the very right thing to do. I18N is about the user-frontend. The specter-object is on a very different layer. I hope we'll be able to keep it that way. Might get challenging, though.
Great idea!